lms/static/scripts/frontend_apps/components/BasicLTILaunchApp.js

```diff
@@ -246,9 +246,20 @@ export default function BasicLTILaunchApp() {
     }
   }, [authToken, canvas.speedGrader, contentReady]);
 
+  // Report a submission only after the first qualifying annotation activity
+  // (create or update an annotation or reply)
   useEffect(() => {
-    reportSubmission();
-  }, [reportSubmission]);
+    const unsubscribe = () =>
+      clientRPC.off('annotationActivity', onAnnotationActivity);
+
+    function onAnnotationActivity() {
+      reportSubmission().then(unsubscribe);
```

Comment:
Given the consequences of this API call failing - it may look to the teacher as though a student has not submitted any work, I think we should catch any failure here and handle it in a more visible way.